chapters/en/slides/module1/module1_00.md

```diff
@@ -4,23 +4,15 @@ type: slides
 
 # Module Learning Outcomes
 
-Notes: In this module we will be learning about the importance of data visualization and how a grammar of graphics can help us effectively visualize data.
+In this module you will learn how to
 
----
```

Comment:
I think we should put back `---` to have the bullet points appear in the next slide instead of the title slide? @joelostblom  I've made a commit to fix this and will push if you agree.

Reply:
Yup please do, that sounds like a mistake on my side!